definitions.yml

```
@@ -427,6 +431,19 @@ definitions:
       enabled:
         type: boolean
         description: "If true paypal is available in those app that support it"
+  CgnConfig:
+    type: object
+    description: "dedicated config for the CGN app flow"
+    required:
+      - enabled
+      - cgn_merchants_v2
+    properties:
+      enabled:
+        type: boolean
+        description: "If true CGN app components and logic is available in those app that support it"
+      cgn_merchants_v2:
```

```suggestion
      merchants_v2:
```
since we already are in `cgn` section
it's just syntax sugar. What do you think?